### PR TITLE
generate a Firefox version

### LINKED ISSUE #29: Throw Firefox some love
Firefox users would appreciate being able to use this too.

### PR DESCRIPTION
**Pull request by MarcBernard**
Resolves #29.

### CODE REVIEW

File: Extension/background.js [hunk 1]
```diff
@@ -41,23 +43,43 @@ function snagHtmlLink(pageDetails) {
...
-              urlMatches: 'jiraprod.agfahealthcare.com/browse/[A-Z][A-Z0-9]+-[0-9]+'
-            },
-          })
-        ],
-        // And shows the extension's page action.
-        actions: [new chrome.declarativeContent.ShowPageAction()]
-      }
-    ]);
-  });
+  if (chrome.declarativeContent) {
```
Comment (MarcBernard) [inline Extension/background.js:46]: Firefox doesn't have declarativeContent, but we still get in here. We'll handle this differently for Firefox, on line 68.

File: build.ps1 [hunk 2]
```diff
@@ -9,7 +9,7 @@ function BuildExtension($manifestFileName, $extensionName) {
     $output = "artifacts\output\$($extensionSlug).zip"
     
     Write-Output "Zipping extension to $output..."
-    .\BuildTools\7z a "$output" .\Extension | ForEach-Object { "  " + $_ }
+    .\BuildTools\7z a "$output" .\Extension\*
```
Comment (MarcBernard) [inline build.ps1:12]: Firefox doesn't like the subfolder in the zip, and Chrome doesn't seem to care.

**Comment by MarcBernard** [comment]
In Firefox...

1.  open about:debugging#/runtime/this-firefox 
2. Click Load Temporary Add-on...
3. Choose any file from the unzipped extension.
4. Navigate to any http://jiraprod.agfahealthcare.com/browse/IEI-* issue.

The extension icon appears in the address bar.

**Review by blairconrad — APPROVED** [review]
Hey, @MarcBernard. You've left the PR on draft for 3 weeks or so, so I didn't know if you were planning more changes. Maybe you're taking a break for the holidays?

I'm not judge of the FF changes, but they _look_ okay, and I trust you.

**Comment by blairconrad** [comment]
Fun fact: I don't get a notification when the PR status changes.

Thanks for your efforts, @MarcBernard. Enjoy your extension!

**Pull request merged**